reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md

```diff
@@ -655,6 +670,23 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SkipCertificateCheck
+Skips certificate validation checks.
+
+This includes all validations such as expiration, revocation, trusted root authority, etc.
```

Comment:
add disclaimer above here as well

Reply:
Done. Also moved the section to the correct location alphabetically with in doc.